src/aap_eda/core/migrations/0023_add_default_organization.py

```diff
@@ -0,0 +1,76 @@
+from django.conf import settings
+from django.db import migrations
+from django.utils import timezone
+
+
+def create_default_org(apps, schema_editor):
+    Organization = apps.get_model("core", "Organization")  # noqa: N806
+
+    db_alias = schema_editor.connection.alias
+
+    now = timezone.now()
+    Organization.objects.using(db_alias).create(
+        name=settings.DEFAULT_ORGANIZATION_NAME,
+        description="The default organization",
+        created_on=now,
+        modified_on=now,
+    )
+
+
+def delete_default_org(apps, schema_editor):
+    Organization = apps.get_model("core", "Organization")  # noqa: N806
+    db_alias = schema_editor.connection.alias
+
+    Organization.objects.using(db_alias).filter(
+        name=settings.DEFAULT_ORGANIZATION_NAME
```

Comment:
umm... wouldn't this cascade delete everything?

Reply:
Yeah it would.. That's another thing I'd like clarity on: how should org foreign key field be defined in models, in terms on_delete, or having CASCADE delete there is fine but we fix this reverse function to loop through all existing objects to set their org to null then delete the default org.

Reply:
For the reverse migration, I suspect it might work better to do nothing. Then it drops the field, but that doesn't cascade delete anything.